editor/editor_system.cpp

```diff
@@ -318,7 +318,10 @@ void EditorSystem::drawDefaultUI(float deltaMilliseconds)
 				ImGui::Separator();
 				if (ImGui::BeginMenu("Create Scene"))
 				{
+					static unsigned int inputid;
```

Comment:
No need to make this static.
Also, the type should be `SceneID` and follow naming conventions `inputID`

Reply:
Imgui takes in argument of type unsigned int. Directly giving as SceneID seemed not to be compatible with imgui